packages/wallets/package.json

```diff
@@ -24,6 +24,7 @@
         "react-dom": "^17.0.2",
         "react-joyride": "^2.5.3",
         "react-router-dom": "^5.2.0",
+        "react-transition-group": "4.4.2",
```

Comment:
How is this library used in the dynamic leverage animation? Is it for flipping the card? 👀

Reply:
Yes, I am using this library for Modal Transition.

Reply:
echoing comments from other PRs - unless its strictly necessary, we shouldn't add new libraries, if its only possible, should be done with css animations,
espcially given the fact that this one is 244kB 😲😲😲

Reply:
Updated the implementation to use CSS animation, not using this library anymore.